Bottom: d5c988584ef608b1811c5e2dd4d96c74a9faf96c
Top:    637903e6c13ffaf37c1e9be7ac68b84cb6b6274e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-10 17:16:17 +0000

Revert "ActionDisplay: show help on failure."

This reverts commit 0978828fcf27be98d4edee22e57a13756a684b62.




---

diff --git a/src/action.c b/src/action.c
index 0be31ab..58adeb6 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2961,13 +2961,11 @@ ActionDisplay (int argc, char **argv, Coord childX, Coord childY)
 	  break;
 	}
     }
-  else
-    err = 1;
 
-  if (err)
-    AFAIL (display);
+  if (!err)
+    return 0;
 
-  return 0;
+  AFAIL (display);
 }
 
 /* --------------------------------------------------------------------------- */
